db/migrate/20160829175601_add_content_filter_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddContentFilterToUser < ActiveRecord::Migration
+  def change
+    add_column :users, :content_filter, :string
```

Comment:
I'm ok with a serialized hash (but not thrilled by it).  However, if we really expect folks to use this in more complex cases, then varchar 255 seems kind of small and it's not clear what would happen if the serialized hash happens to be more than 255 characters long.  Maybe this should be a text field like the other fields in User.